perftracker/views.py

```diff
@@ -509,11 +509,7 @@ class RegressionJson(BaseDatatableView):
         max_display_length = 5000
 
         def render_column(self, row, column):
-            # We want to render user as a custom column
-            if column == 'tests_total':
-                return '{0} {1}'.format(row.tests_total, row.tests_completed)
-            else:
-                return super(JobJson, self).render_column(row, column)
+            return super(RegressionJson, self).render_column(row, column)
```

Comment:
Hm. Nice catch, but the proper fix would be to just remove this method (so the default render_column will be called)